eng/common/scripts/Package-Properties.ps1

```diff
@@ -205,4 +221,21 @@ function Get-PkgListFromYml ($ciYmlPath)
         LogError "Failed to retrive package names in ci $ciYmlPath"
     }
     return $artifactsInCI
-}
\ No newline at end of file
+}
+
+function Install-PowershellYamlModule
```

Comment:
I'm not sure based on lack of context where this script will get used, but since you're refactoring here I wonder if it would be better to have the yaml install as a separate step that we can re-use throughout the pipelines.

Reply:
I'd actually like to see what options we have to just pull in the source for this module and using it directly instead of installing it because the install seems to cause reliability issues in various pipelines.

Reply:
The module source is quite small, it's just a wrapper around YamlDotNet. The former is apache licensed, the latter MIT licensed. I forget if apache 2.0 is good/bad for copying?

Reply:
We should consider just wrapping YamlDotNet ourselves assuming we can come up with a good way to get and load the binary.